pkg/apis/logging/v1alpha1/types.go

```diff
@@ -122,8 +131,16 @@ type ElasticsearchStatus struct {
 }
 
 type CollectionStatus struct {
-	FluentdStatus    FluentdCollectorStatus `json:"fluentdStatus,omitempty"`
-	NormalizerStatus NormalizerStatus       `json:"normalizerStatus,omitempty"`
+	LogCollection    LogCollectionStatus   `json:"logCollection,omitempty"`
+	EventCollection  EventCollectionStatus `json:"eventCollection,omitempty"`
+	NormalizerStatus NormalizerStatus      `json:"normalizerStatus,omitempty"`
+}
+
+type LogCollectionStatus struct {
+	FluentdStatus FluentdCollectorStatus `json:"fluentdStatus,omitempty"`
```

Comment:
Same here - we'll have both FluentdStatus and RsyslogStatus?

Reply:
Correct